packages/actions/src/Call/executeCall.spec.ts

```
@@ -128,4 +128,172 @@ describe('executeCall', () => {
 		expect(result.errors[0].code).toBe(-32012)
 		expect(result.errors[0].message).toMatchSnapshot()
 	})
+
+	describe('async storage prefetch', () => {
+		// Mock AbortController
+		let mockAbortController
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)